### PR TITLE
done feat:auth-ui

### LINKED ISSUE #4: Register
- for user
- for expert

### LINKED ISSUE #5: Login
- save the token into localStorage
- handle refresh token

### PR DESCRIPTION
**Pull request by HxYNQ2002**
closes #4 
closes #5

### CODE REVIEW

File: src/components/SigninForm/SigninForm.jsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+import backgroundImage from 'Images/heroSen.jpg'
+import PersonIcon from '@mui/icons-material/Person'
+import LockRoundedIcon from '@mui/icons-material/LockRounded'
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked'
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+
+const InputText = styled(TextField)(() => ({
+    outline: '0.1rem solid #69AD28',
+    borderRadius: '0.5rem',
+}))
```
Comment (buitannguyen2612) [inline src/components/SigninForm/SigninForm.jsx]: You need write this into another file example: SigninForm.styled.js, and then import to use it

File: src/components/SigninForm/SigninForm.jsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                sx={{
+                    position: 'relative',
+                    backgroundImage: `url(${backgroundImage})`,
+                    backgroundPosition: 'center',
+                    backgroundSize: 'cover',
+                    backgroundRepeat: 'no-repeat',
+                    width: '100vw',
+                    height: '100vh',
+                    filter: `blur(5px)`,
+                    zIndex: -1,
```
Comment (buitannguyen2612) [inline src/components/SigninForm/SigninForm.jsx]: There is alot of sx style here, like i say in first comment, you need to add another styled file and then import to use.

File: src/components/SigninForm/SigninForm.jsx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+                sx={{
+                    position: 'absolute',
+                    top: '10rem',
+                    left: '53rem',
+                    textAlign: 'center',
+                    fontSize: '2rem',
+                    fontWeight: '900',
+                    color: '#FFF',
+                }}
+            >
```
Comment (buitannguyen2612) [inline src/components/SigninForm/SigninForm.jsx]: its same like first comment, reuse it in another component

File: src/components/SigninForm/SigninForm.styled.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
+const {
+    styled,
+    TextField,
+    Box,
+    Typography,
+    Button,
+    Link,
+} = require('@mui/material')
```
Comment (buitannguyen2612) [inline src/components/SigninForm/SigninForm.styled.js]: i think you should use (import ... from...) instead this

File: src/components/SigninForm/SigninForm.jsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    return (
+        <styleMui.Form
+            sx={{
+                position: 'absolute',
+                top: '5rem',
+                left: '10rem',
+                backgroundColor: '#fff',
+                width: '25rem',
+                height: '36rem',
+                borderRadius: '1rem',
```
Comment (buitannguyen2612) [inline src/components/SigninForm/SigninForm.jsx]: why this style of sx still here, its must stay in styled.js file right ?

File: src/components/SigninForm/SigninForm.jsx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+                        ),
+                    }}
+                />
+            ))}
+            <Box
+                sx={{
+                    marginLeft: '3.5rem',
+                    marginTop: '1rem',
+                    display: 'flex',
+                    flexWrap: 'nowrap',
```
Comment (buitannguyen2612) [inline src/components/SigninForm/SigninForm.jsx]: this style should stay in styled file

File: src/components/SignupForm/SignupForm.jsx [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+                            ),
+                        }}
+                    />
+                ))}
+                <Box
+                    sx={{
+                        marginLeft: '3.5rem',
+                        marginTop: '1rem',
+                        display: 'flex',
+                        flexWrap: 'nowrap',
```
Comment (buitannguyen2612) [inline src/components/SignupForm/SignupForm.jsx]: put it into styled file

File: src/components/SignupForm/SignupForm.styled.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
+const {
+    styled,
+    TextField,
+    Box,
+    Typography,
+    Button,
+    Link,
+} = require('@mui/material')
```
Comment (buitannguyen2612) [inline src/components/SignupForm/SignupForm.styled.js]: use another import, for example :  import { Box, styled } from '@mui/material'

File: src/pages/SigninPage/SigninPage.jsx [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+                variant="h5"
+                sx={{
+                    position: 'absolute',
+                    top: '16rem',
+                    left: '51rem',
+                    textAlign: 'center',
+                    fontSize: '1.5rem',
+                    fontWeight: '200',
+                    fontStyle: 'italic',
+                    color: '#FFF',
```
Comment (buitannguyen2612) [inline src/pages/SigninPage/SigninPage.jsx]: put it into styled file

File: src/pages/SigninPage/SigninPage.jsx [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+            <Typography
+                variant="h5"
+                sx={{
+                    position: 'absolute',
+                    top: '10rem',
+                    left: '53rem',
+                    textAlign: 'center',
+                    fontSize: '2rem',
+                    fontWeight: '900',
+                    color: '#FFF',
```
Comment (buitannguyen2612) [inline src/pages/SigninPage/SigninPage.jsx]: put it into styled file

File: src/pages/SigninPage/SigninPage.jsx [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+                    position: 'relative',
+                    backgroundImage: `url(${backgroundImage})`,
+                    backgroundPosition: 'center',
+                    backgroundSize: 'cover',
+                    backgroundRepeat: 'no-repeat',
+                    width: '100vw',
+                    height: '100vh',
+                    filter: `blur(5px)`,
+                    zIndex: -1,
+                }}
```
Comment (buitannguyen2612) [inline src/pages/SigninPage/SigninPage.jsx]: put it into styled file

File: src/pages/SignupPage/SignupPage.jsx [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+                sx={{
+                    position: 'relative',
+                    backgroundImage: `url(${backgroundImage})`,
+                    backgroundPosition: 'center',
+                    backgroundSize: 'cover',
+                    backgroundRepeat: 'no-repeat',
+                    width: '100vw',
+                    height: '100vh',
+                    filter: `blur(5px)`,
+                    zIndex: -1,
```
Comment (buitannguyen2612) [inline src/pages/SignupPage/SignupPage.jsx]: put it into styled file

File: src/pages/SignupPage/SignupPage.jsx [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+            <Typography
+                variant="h5"
+                sx={{
+                    position: 'absolute',
+                    top: '10rem',
+                    left: '53rem',
+                    textAlign: 'center',
+                    fontSize: '2rem',
+                    fontWeight: '900',
+                    color: '#FFF',
```
Comment (buitannguyen2612) [inline src/pages/SignupPage/SignupPage.jsx]: put it into styled file

File: src/components/SignupForm/SignupForm.jsx [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+export default function SignupForm() {
+    const [value, setValue] = React.useState('người dùng')
+
+    const handleChange = (event, newValue) => {
+        setValue(newValue)
+    }
+
+    const nabItem = [
+        {
+            lable: 'người dùng',
```
Comment (phunlh2001) [inline src/components/SignupForm/SignupForm.jsx]: correctly is the label not lable

**Review by phunlh2001 — CHANGES_REQUESTED** [review]
Please fix this

**Pull request merged**